src/mixins/document-mixin.ts

```diff
@@ -78,8 +79,8 @@ export default class DocumentMixin extends Vue {
       const pdf = await pdfjsLib.getDocument({ data: pdfData })
       const perms = await pdf.getPermissions()
       return { isEncrypted: false, isContentLocked: !!perms }
-    } catch (e) {
-      if (e.name === 'PasswordException') {
+    } catch (error) {
+      if (error['name'] === 'PasswordException') {
```

Comment:
`e` or `error` are of type "unknown", so they can't be de-referenced using dot notation. But this works without causing a TS/lint error.

Reply:
good call!